tests/unit/sign-helper.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { parseMessageToSign } from "@/helpers/SignHelper";
+import { parsedMessage } from "../mocks/parsedMessage";
+
+describe("SignHelper", () => {
+  it("Should create the parsed message structure", () => {
+    const messageParsed = parseMessageToSign(
```

Comment:
you're verifying the structure, or the content? or both? despite the test naming is confusing, the test should be verifying that the structure matches, in a more exhaustive way.

Reply:
The structure and content. Based on what the function receive as parameter this test will validate if the structure will be returned correctly.